react-native-pokedex/src/components/card.jsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { View, Text, StyleSheet, Image } from "react-native";
+import { useFonts } from "expo-font";
+
+
+const Card = (props) => {
+  const [loaded] = useFonts({
+    Montserrat: require('./assets/fonts/Montserrat.ttf'),
```

Comment:
I get an error on load saying there is no Montserrat.ttf at that path. For me the path is '../../assets/fonts/Montserrat.ttf'